.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
```

Comment:
The code changes involve updating the formatting and content of the `RELEASE_NOTES` variable. The changes include removing unnecessary characters, adjusting line breaks, and modifying the way certain elements are displayed. The reviewer should ensure that the changes do not introduce any errors and that the formatting and content improvements are appropriate for the context in which the variable is used.